ruling/index.ts

```diff
@@ -25,7 +25,7 @@ import lodash from 'lodash';
 import minimist from 'minimist';
 import sonarjs from '../src';
 
-const rulesPath = path.join(__dirname, '../lib/rules');
+const rulesPath = path.join(__dirname, '../lib/src/rules');
```

Comment:
Out of curiosity, how was it working before this change?

Reply:
This is due to the "resolveJSONModule": true in the tsconfig. I think it's due to this, we need to pull in the package.json into the generated 'lib' folder and everything is then a level deeper within 'lib/src'
before: 'lib/rules', 'lib/utils' <- no package.json
after: 'lib/package.json', 'lib/src/rules', 'lib/src/utils'